manifests/0000_90_dns-operator_03_prometheusrules.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: dns
+  namespace: openshift-dns-operator
+  labels:
+    role: alert-rules
+spec:
+  groups:
+    - name: openshift-dns.rules
+      rules:
+      - alert: CoreDNSPanicCount
+        expr: coredns_panic_count_total > 0
+        for: 5m
+        labels:
+          severity: warning
+        annotations:
+          message: |
+            "{{ $value }} CoreDNS panics encountered on {{ $labels.instance }}"
```

Comment:
Using `""` with `|` means you're quoting a quoting string; is that necessary?

Reply:
It is not, will fix.